Jenkinsfile

```diff
@@ -18,8 +18,9 @@ node {
           sh 'make compose-test'
         }
 
-        stage('Build & push kuma image') {
+        stage('Build & push images') {
           sh 'make build-kuma push-kuma'
+          sh 'make push-base VERSION=latest'
```

Comment:
@jgmize I'm 50% sure this is wrong...

Maybe it push both ``kuma_base:<commit>`` and ``kuma_base:latest``:

```
sh 'make build-kuma push'
sh 'make push-base VERSION=latest'
```

Maybe branches should also push commit-tagged kuma_base images.